### PR TITLE
square-event.md

### PR DESCRIPTION
**Pull request by bdamerac**
Adds square workshop details

### CODE REVIEW

**Comment by evykassirer** [comment]
nice! all in one commit with no merges and stuff

see it here: https://wics-site-pr-208.herokuapp.com/

Can you look at the other events that have been added before and add the same details that are in those files?

**Comment by evykassirer** [comment]
also it'd be good to add the registration link

File: content/S2016/square-event.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Tags: academic 
+Slug: square-event-s16 
+Author: Bhargavi Dameracharla 
+Summary: Technical workshop on open source HTTP & HTTP/2 client
+
+Square is holding a technical workshop on their open source HTTP & HTTP/2 client for Android and Java applications. 
+Join us to learn more about Okhttp and get a chance to speak with their engineers! Food will be available. 
+(Including vegetarian, vegan, and gluten-free options)
+
+Note: There is an attendance cap of 70 people (50 undergrads and 20 grads), so make sure to register early!
```
Comment (ehashman) [inline content/S2016/square-event.md:13]: This is missing an event details section. Can you copy this from another event?

Comment (ehashman) [inline content/S2016/square-event.md:13]: Also, there is no registration link. Can you add that?

Also, is this event women-only or open to the public?

File: content/S2016/square-event.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+title: Square Tech Workshop: Becoming Ok with Okhttp
+Date: 2016-05-25
```
Comment (ehashman) [inline content/S2016/square-event.md:2]: This is the date of the website post, not the date of the event. We should make sure to include the event date somewhere in here and update this to reflect today's date.

File: content/S2016/square-event.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+title: Square Tech Workshop: Becoming Ok with Okhttp
+Date: 2016-05-25 
+Category: Events 
+Tags: academic
```
Comment (ehashman) [inline content/S2016/square-event.md:4]: I'd say the tags "industry", "workshop", and "open source" more accurately describe this event :) You can take a look at all the current tags attached to posts at http://wics.uwaterloo.ca/tags/ .

**Comment by bdamerac** [comment]
Okay I'll add all the things mentioned here. Thanks for your patience :)

**Comment by ehashman** [comment]
Hi @Bhargavi1997,

You can address these things by adding commits to your current branch to address the feedback. You don't need to reopen a new pull request! (and indeed, it's actually easier to track if things are addressed if you make the commits here.)

**Comment by ehashman** [comment]
Oh wait, I see... does "unknown repository" mean that you were editing this on the github web UI?

**Comment by ehashman** [comment]
Superseded by #209 apparently.

**Pull request closed without merging**